hc/front/views.py

```diff
@@ -60,6 +60,28 @@ def my_checks(request):
     return render(request, "front/my_checks.html", ctx)
 
 
+@login_required
+def failed_checks(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
```

Comment:
We talked about this, use descriptive variable names

Reply:
Our bad we are changing the code...